src/pymor/models/interface.py

```diff
@@ -222,7 +223,7 @@ def _compute_solution_error_estimate(self, solution, mu=None, **kwargs):
             raise ValueError('Model has no error estimator')
         return self.error_estimator.estimate_error(solution, mu, self, **kwargs)
 
-    def _compute_output_error_estimate(self, solution, mu=None, **kwargs):
+    def _compute_output_error_estimate(self, solution, mu=None, return_vector=False, **kwargs):
```

Comment:
There should probably be a doc entry for `return_vector` below.

Reply:
Would `return_array` be a more sensible name, given that we have `output_d_mu_return_array` below in `compute`?

Reply:
definitely!

Reply:
I disagree regarding `return_vector` vs. `return_array`: Both parameters do completely different things. `return_array` is used to determine the *format* of the return value as a Python object. `return_vector` is used to determine *different mathematical quantities* that are to be computed, name a vector of estimates vs. a norm of this vector.

Reply:
Thanks @sdrave, This is probably a reasonable argument for keeping the name `vector` as it is.